### PR TITLE
[12.0][FIX] excel_import_export, use sudo() to get param

### PR DESCRIPTION
**Pull request by kittiu**
Small fix. Just to ensure that the param path_temp_file is readable.

### CODE REVIEW

File: excel_import_export/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     'name': 'Excel Import/Export',
     'summary': 'Base module for easy way to develop Excel import/export',
-    'version': '12.0.1.0.1',
+    'version': '12.0.1.0.2',
```
Comment (pedrobaeza) [inline excel_import_export/__manifest__.py]: FYI, merge bot does this, for avoid possible conflicts, so no need to do it yourself

File: excel_import_export/models/xlsx_export.py [hunk 2]
```diff
@@ -237,7 +237,7 @@ def export_xlsx(self, template, res_model, res_id):
         # Prepare temp file (from now, only xlsx file works for openpyxl)
         decoded_data = base64.decodestring(template.datas)
         ConfParam = self.env['ir.config_parameter']
```
Comment (pedrobaeza) [inline excel_import_export/models/xlsx_export.py]: put `sudo()` here

File: excel_import_export/readme/HISTORY.rst [hunk 3]
```diff
@@ -3,7 +3,12 @@
 
 * Start of the history
 
-12.0.1.0.2 (2019-06-24)
+12.0.1.0.1 (2019-06-24)
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 * Fix wizard on v12 can't download sample template file - https://github.com/OCA/server-tools/issues/1574
+
+12.0.1.0.2 (2019-08-07)
```
Comment (pedrobaeza) [inline excel_import_export/readme/HISTORY.rst]: This should be ordered the other way.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by kittiu** [comment]
@pedrobaeza thank you for the review. Fixed.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I'm seeing a hole here due to changelog: if you don't put version number and this one gets merged after another PR for the same module, the changelog numbering won't be correct (and that's another reason why I don't like static changelog maintenance having commit history).

@sbidoul how do you workaround this problem?

**Comment by sbidoul** [comment]
@pedrobaeza about changelogs yes, there is nice tool named [towncrier](https://pypi.org/project/towncrier/) which basically lets you define changelog entries as small files of the form `<issue/pr number>.(bugfix|feature|...)`. You add such files in the PRs and when releasing towncrier generates the changelog entries (with a nice backlink to the issue/pr)  and remove those files.

I'm experimenting it on [mis builder](https://github.com/OCA/mis-builder/blob/10.0/mis_builder/readme/HISTORY.rst), and it is somehting we could at some point consider generalizing, with proper OCA tooling such as bot support.

**Comment by pedrobaeza** [comment]
OK, that can be the solution then.

**Comment by pedrobaeza** [comment]
Not sure why it's failing... let's rety again

/ocabot merge patch

**Comment by pedrobaeza** [comment]
No clue what's happening. @kittiu can you rebase this PR to see?

**Comment by sbidoul** [comment]
Looks like travis is quite unstable these days. I've restarted the failed travis job.

**Comment by kittiu** [comment]
@pedrobaez is this ok now to quick merge?
Note: I am working on including the feature from report_excel to excel_import_export.
Thank you!

**Comment by pedrobaeza** [comment]
Let's merge it manually

File: excel_import_export/__manifest__.py (hunk 1 above)
Comment (kittiu) [reply excel_import_export/__manifest__.py]: So, when it get merged manually, the version is not updated (I will update to .2 manually, hoping the bot will update to .3).
Just FYI.

**Pull request merged**